source/js/payments-card.js

```diff
@@ -15,11 +15,23 @@ function setupBraintree() {
       "An error occurred. Please reload the page or try again later."
     ),
     errorDiv = document.getElementById("payments__braintree-errors-card"),
+    yourDetailsFormErrorDiv = document.getElementById(
```

Comment:
Div at the top of the form that will render saying "there were some errors with your form. Please correct the highlighted fields" if necessary